service_mesh/v2x/ossm-vs-community.adoc

```diff
@@ -5,21 +5,19 @@ include::modules/ossm-document-attributes.adoc[]
 
 toc::[]
 
-An installation of {ProductName} differs from upstream Istio community installations in multiple ways. The modifications to {ProductName} are sometimes necessary to resolve issues, provide additional features, or to handle differences when deploying on {product-title}.
-
-The current release of {ProductName} differs from the current upstream Istio community release in the following ways:
+{ProductName} differs from an installation of Istio to provide additional features or to handle differences when deploying on OpenShift.
```

Comment:
s/OpenShift/{product-title}

Reply:
@bmcelvee Thanks! knocked that out and squashed. Ready to merge and cp to 4.6-4.8 when you are.